### PR TITLE
feat(android): implement borderRadius corner support

### PR DESCRIPTION
**Pull request by garymathews**
- Implement the ability to specify radius for specific `View` corners

##### TEST CASE
```JS
const window = Ti.UI.createWindow({ backgroundColor: 'white' });
const view_a = Ti.UI.createView({
	backgroundColor: 'red',
	width: 100,
	height: 50,
	top: 100,
	borderRadius: '20 20px 0 0'
});
const view_b = Ti.UI.createView({
	backgroundColor: 'blue',
	width: 100,
	height: 50,
	top: 150,
	borderRadius: [ '0', '0px', '40dp', 20 ]
});

window.add([ view_a, view_b ]);
window.open();
```

[JIRA Ticket](https://jira.appcelerator.org/browse/TIMOB-27946)

### CODE REVIEW

File: apidoc/Titanium/UI/View.yml [hunk 1]
```diff
@@ -1332,8 +1332,11 @@ properties:
   - name: borderRadius
     summary: Radius for the rounded corners of the view's border.
     description: Each corner is rounded using an arc of a circle.
-    type: Number
+        Values for each corner can be specified. For example, '20px 20px' will set both left and right corners to `20px`.
+        Specifying '20px 20px 20px 20px' will set top-left, top-right, bottom-right and bottom-left corners in that order.
+    type: [Number, String, Dictionary<Number>, Dictionary<String>]
```
Comment (sgtcoolguy) [inline apidoc/Titanium/UI/View.yml:1337]: I think this is meant to be:
```suggestion
    type: [Number, String, Array<Number>, Array<String>]
```

**Review by sgtcoolguy — APPROVED** [review]
changes LGTM. Ideally we'd have an actual UI test for this, but that requires us to merge #11699 and get the view comparisons code in the test suite first.

Only change I see is the API doc mismatch (expecting `Dictionary` instead of `Array`)

So this is approved with the caveat that the API doc change needs to be fixed before merge.

**Review by ssjsamir — APPROVED** [review]
**FR Passed** Tested with the test case above, able to see `borderRadius corner` change the border radius control also tested with different value e.g.

```
const view_a = Ti.UI.createView({
	backgroundColor: 'red',
	width: 100,
	height: 50,
	top: 100,
	borderRadius: '20 20px 20 30'
});
const view_b = Ti.UI.createView({
	backgroundColor: 'blue',
	width: 100,
	height: 50,
	top: 150,
	borderRadius: [ '20', '30px', '40dp', 20 ]
});

```
**Test Environment**
```
MacOS Big Sur: 11.0 Beta
Xcode: 12.0 Beta 
Java Version: 1.8.0_242
Android NDK: 21.3.6528147
Node.js: 12.18.1
""NPM":"5.0.0","CLI":"8.0.0""
API29 Pixel XL emulator
```

**Comment by sgtcoolguy** [comment]
I'll combine this with #11801 and add a few more tests/UI snapshot tests...

**Comment by sgtcoolguy** [comment]
manually merged into master (after fixing bug with applying 2 value use case against wrong corners)

**Comment by ssaddique** [comment]
Fix verified on build 9.1.0.v20200804082025.

**Test environment**
```
OS Ver:         10.15.3
Xcode Ver:      Xcode 11.6
Appc NPM:       5.0.0
Appc CLI:       8.0.0
Node Ver:       10.17.0
Java Ver:       1.8.0_162
Emulators:      Pixel 3 (API 26, 27, 28, 29, 30)
```

**Pull request closed without merging**